python/py-pacal/Portfile

```diff
@@ -17,11 +17,12 @@ homepage            http://pacal.sourceforge.net/index.html
 master_sites        pypi:P/PaCal/
 distname            PaCal-${version}
```

Comment:
under the "name" field, please add:
"python.rootname PaCal"

then you can remove "distname" and "master_sites"